lms/static/sass/discussion/views/_thread.scss

```diff
@@ -25,21 +39,13 @@ body.discussion, .discussion-module {
 
   // response layout
   .discussion-response {
-    min-height: ($baseline*7.5);
-
-    .username {
-      @include font-size(14);
-      @extend %t-weight5;
-    }
-
-    .posted-details .username {
-      font-size: inherit;
-    }
+    min-height: ($baseline*5);
 
     .response-header-content {
       display: inline-block;
       vertical-align: top;
-      width: flex-grid(11,12);
+      // width: calc((flex-grid(11,12)) - $response-image-dimension);
```

Comment:
Is this needed still (since its commented out and you have a ``flex-grid``-based width)?